buildconfig/appveyor.yml

```diff
@@ -168,8 +168,7 @@ install:
   - "%PYTHON% -m pip install Sphinx==4.5.0 sphinxcontrib-applehelp==1.0.2 pip install sphinxcontrib-devhelp==1.0.2 sphinxcontrib-htmlhelp==2.0.0 sphinxcontrib-qthelp==1.0.3 sphinxcontrib-serializinghtml==1.1.5"
   - "%PYTHON% setup.py docs"
   - "%WITH_COMPILER% %PYTHON% -m buildconfig"
-  - "%WITH_COMPILER% %PYTHON% setup.py build -j4 %USE_ANALYZE%"
-  - "%WITH_COMPILER% %PYTHON% setup.py -setuptools %DISTRIBUTIONS%"
+  - "%WITH_COMPILER% %PYTHON% -m pip wheel . --wheel-dir=dist"
```

Comment:
It would be better if the "-j4" part was passed in. However, it's not the end of the world, because it doesn't seem to help as much as it used to help. Now it only saves 2-4 minutes total off the build time.